src/test/regress/sql/foreign_tables_mx.sql

```diff
@@ -219,14 +227,21 @@ SELECT * FROM ref_tbl d JOIN foreign_table_local f ON d.a=f.id ORDER BY f.id;
 
 SET search_path TO foreign_tables_schema_mx;
 
--- should error out because doesn't have a table_name field
 CREATE FOREIGN TABLE foreign_table_local_fails (
         id integer NOT NULL,
         data text
 )
         SERVER foreign_server_local
         OPTIONS (schema_name 'foreign_tables_schema_mx');
 
+-- should error out because doesn't have a table_name field
+SELECT citus_add_local_table_to_metadata('foreign_table_local_fails');
+
+-- should work since it has a table_name
+ALTER FOREIGN TABLE foreign_table_local_fails OPTIONS (table_name 'foreign_table_local_fails');
```

Comment:
NIT: Can we change table_name to something different than 'foreign_table_local_fails' which is also foreign table's name. There is no query for the table but that created a self loop and consumed all connections on my local.